prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html

```diff
@@ -40,23 +40,34 @@
 
         </div>
 
-        <ng-container *ngIf="hasAttemptedSubmission && (remainingAttempts === 1 || locked)">
+        <ng-container *ngIf="hasAttemptedSubmission && (unauthorized || remainingAttempts === 1 || locked)">
           <app-alert type="danger"
                      icon="error_outline">
             <ng-container #alertContent
                           class="alert-content">
-              <ng-container *ngIf="locked; else oneMoreAttempt">
-                Your account is now locked due to too many failed login attempts. If you think this is a mistake please
-                contact Reg Admin at <a href="tel:12509529137">(250) 952-9137</a> or
-                <a href="mailto:HLTH.REGISTRIESADMIN@gov.bc.ca">HLTH.REGISTRIESADMIN@gov.bc.ca</a>.
+              <ng-container *ngIf="unauthorized; else authenticationFailed">
+                You have been authenticated, but do not sufficient permission to proceed. If you think this is a
```

Comment:
add "have" ie 
`You have been authenticated, but do not have sufficient permission to proceed. If you think this is a`